RegressionTests.hs

```diff
@@ -0,0 +1,94 @@
+{-# LANGUAGE OverloadedStrings #-}
+module RegressionTests where
+
+import Database.HDBC.Sqlite3
+import Database.HDBC
+import System.IO
+import System.Environment
+import System.Exit
+import System.Process
+import Data.Aeson
+import Data.Aeson.Types
+import Data.Maybe
+import qualified Data.ByteString.Lazy.UTF8 as BLU
+import qualified Data.HashMap.Strict as HM
+import Data.Text (Text)
+import qualified Data.Text.IO as TIO
+import Data.List
+
+status_list :: [Text]
+status_list = ["", "syntaxerror", "notequiv", "similar", "correct", "expected", "buggy"]
+
+getScore :: Maybe Text -> Int
+getScore status = fromMaybe 0 $ elemIndex (fromMaybe "" status) status_list
+
+loadFromDB :: String -> String -> IO [[SqlValue]]
+loadFromDB dbPath sql = do
+    conn <- connectSqlite3 dbPath
+    dat <- quickQuery' conn sql []
+    disconnect conn
+    return dat
+
+getInputOutput :: String -> IO [(String, String)]
+getInputOutput database = do
+  results <- loadFromDB database "select input, output from requests where service = 'diagnoser';"
```

Comment:
It looks like you never use the `output` values in the regression test. Perhaps not select them at all?